src/sentry/static/sentry/app/views/discover/result/pagination.tsx

```diff
@@ -16,7 +16,7 @@ export default class Pagination extends React.Component<PaginationProps> {
     const {getPreviousPage, getNextPage, previous, next} = this.props;
 
     return (
-      <PaginationButtons className="btn-group">
```

Comment:
@markstory is it ok to break this? I don't even know how to test discoverv1 anymore

Reply:
https://storage.googleapis.com/sentry-visual-snapshots/getsentry/sentry/9a7d14a7edcd0450f3c116b439aef3c0e170cba6/index.html We could probably wrap this with ButtonBar but since discover is getting removed soon... not sure how important this is

Reply:
NM I updated to use ButtonBar